frontend-new/components/LinkManagementPage.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box } from '@mui/material'
+import { FC } from 'react'
+
+import {
+  ExtensionNotification,
+  LinkCreationForm,
+  LinkList,
+  NoLinksNotification,
+  ResponseContainer,
+  Search,
+} from 'app/components'
+import { useLinkList } from 'app/hooks/links'
+import { media } from 'app/styles/theme'
+
+export const LinkManagementPage: FC = () => {
+  const {
+    notificationState,
+    extensionInstalled,
+    filterValue,
+    setFilterValue,
+    displayLinks,
+    onSave,
+    linksExists,
+    noLinks,
+  } = useLinkList()
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        backgroundColor: '#fff',
+        height: '100%',
+        pt: 4,
+        px: 3,
+        [media.TABLET]: {
+          pt: 8,
+          px: 10,
+        },
+        [media.DESKTOP]: {
+          px: 25,
+        },
+      }}
+    >
+      {!extensionInstalled && <ExtensionNotification />}
+      <Box>
+        <LinkCreationForm onCreate={onSave} />
+        {notificationState && <ResponseContainer {...notificationState} />}
+        {noLinks && <NoLinksNotification />}
+        <Search value={filterValue} onChange={setFilterValue} />
```

Comment:
If there aren't any links should we hide the search box?

Reply:
no, we need to keep search box in any case according to @itsojon